### ISSUE TITLE
Mod incompatibility: Pause Instead of Catching Up

### ISSUE THREAD

**Issue by AstraLuma** [Palats/mapshot#27]
Not sure what can be done about this, but imma drop it here:

It looks like [Pause instead of catching up](https://mods.factorio.com/mod/piocu-pause-instead-of-catching-up) is incompatible with the mapshot CLI.

It causes the process to just stop.

**Comment by Palats** [comment]
Factorio mods are really tied to the sync'ed loop of Factorio - and thus do nothing when in pause as it would risk desync'ing the game. That includes running screenshotting and/or setting out of pause from the script (Rseding91 do [confirms](https://forums.factorio.com/viewtopic.php?p=286978#p286978)).

Given that the CLI do fiddle with a few things, there might be something in that direction, however:
 - Factorio does not seem to have flags to ignore pause (except something for benchmarking, unlikely to be relevant).
 - I imagine pause state is kept in the savegame file, and it is opaque in practice (and Factorio dev do not want to document it / open it, which is understandble).

So, long story short, I don't think there is anything doable here, so, closing unfortunately. (Obv. if you have a specific idea on how to avoid this problem, I'm all ears).

**Comment by AstraLuma** [comment]
The pause state doesn't seem to be saved (at least in this case), and simply removing the mod seems to clear the problem up.

Given that the mapshot cli already copies the mod directory, putting a filter on that process seems relatively easy?

In the short term, I'll just add a script to my pipeline to remove it.

**Comment by Palats** [comment]
Ah, thanks for the clarification - indeed, adding a way to filter out mods on the CLI would be straightforward and potentially useful.
